tests/Linna/Http/Message/UriTest.php

```diff
@@ -146,7 +146,7 @@ public function portProvider(): array
      */
     public function testGetPort(string $scheme, string $port, int $expected): void
     {
-        $this->assertEquals($expected, (new Uri("{$scheme}://username:password@hostname.com{$port}/path?arg=value#anchor"))->getPort());
+        $this->assertSame($expected, (int)(new Uri("{$scheme}://username:password@hostname.com{$port}/path?arg=value#anchor"))->getPort());
```

Comment:
In the data providers, the `set#0` and `set#4` will return `null` when calling the `Uri::getPort` method.

It should casting the `null` to be `0` when asserting the above two test data.